docs/JSON_FORMAT.md

```diff
@@ -69,12 +71,12 @@ _gpiobanks_ describes settings for a gpio bank of the SoC. At the moment only ba
 > #### INFO
 > Only GPIOs that are listed in the JSON will be modified. All other GPIOs will keep their settings.
 
-| Field      | JSON Datatype         | Range     | Description                              |
-|:-----------|:----------------------|:----------|:-----------------------------------------|
-| drive      | string (enum)         | see below | Set drive strength of gpio bank          |
-| slew       | string (enum)         | see below | Set slew rate of gpio bank               |
-| hysteresis | string (enum)         | see below | Set hysteresis of gpio bank              |
-| gpios      | array of gpio objects |           | List of gpios, that should be configured |
+| Field      | Field Type | JSON Datatype         | Range     | Description                              |
+|:-----------|:-----------|:----------------------|:----------|:-----------------------------------------|
+| drive      | mandatory  | string (enum)         | see below | Set drive strength of gpio bank          |
+| slew       | mandatory  | string (enum)         | see below | Set slew rate of gpio bank               |
+| hysteresis | mandatory  | string (enum)         | see below | Set hysteresis of gpio bank              |
+| gpios      | optional   | array of gpio objects |           | List of gpios, that should be configured |
```

Comment:
What happens in the tool, if gpios are defined twice? I see no easy check in the schema to prevent that. How does the tool handle this?

Made `gpios` mandatory, btw. In both the schematic and the documentation.

Reply:
The tool just goes through the list and sets the gpios, so last one wins. We should document the behavior. I'm not sure if we need to give an error or warning if a gpio is assigned twice.